### DISCUSSION TITLE
I have not yet received the benefits of the student pack

### DISCUSSION THREAD

**Discussion by pauldvlp** [community/community#131588] [GitHub Education]
### Hi everyone,

**Good morning dear**

Thank you for accepting me on Github Students, it is a pleasure to be part of this community.

The reason for the discussion is because I have not yet received the benefits of the student package.

![image](https://github.com/community/community/assets/72539686/ceaa4b61-679b-49e1-b6e2-79a627324d0b)

My application was approved on June 28, 2024, thank you.

**Comment by sizan19** [comment]
yea mine too
